src/injected/web/gm-wrapper.js

```diff
@@ -1,4 +1,6 @@
-import { INJECT_PAGE, INJECT_CONTENT, METABLOCK_RE } from '#/common/consts';
+import {
+  INJECT_PAGE, INJECT_CONTENT, INJECT_WRAP, METABLOCK_RE,
```

Comment:
I always wanted to ask, why `3` items is the limit for a single line object literal? I would expect the limit to be higher or even based on the line length.

Reply:
Just default options... I guess one line per item is expected by the lint rule.

Reply:
Well, if it doesn't irritate you I'll just try to get used to it.